### PR TITLE
FSA-123 story review 

### PR DESCRIPTION
**Pull request by lunaoll**

### CODE REVIEW

**Review by atakanozgur1 — APPROVED** [review]

**Review by atakanozgur1 — APPROVED** [review]

File: src/main/java/luna/week1/Test.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package luna.week1;
+
+public class Test {
+
+    //test case 1 added
+
+    //test case 2 added
```
Comment (mery666-ui) [inline src/main/java/luna/week1/Test.java:7]: good method

File: src/main/java/luna/week1/Test.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package luna.week1;
+
+public class Test {
+
+    //test case 1 added
```
Comment (lunaoll) [inline src/main/java/luna/week1/Test.java:5]: variable name does not follow naming convention

File: src/main/java/luna/week1/Test.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+    //test case 2 added
+}
```
Comment (lunaoll) [inline src/main/java/luna/week1/Test.java:8]: we can extract it to a method

File: src/main/java/luna/week1/Test.java (hunk 2 above)
Comment (mery666-ui) [reply src/main/java/luna/week1/Test.java:5]: ok I will

File: src/main/java/luna/week1/Test.java (hunk 3 above)
Comment (mery666-ui) [reply src/main/java/luna/week1/Test.java:8]: ok , nice catch

**Pull request merged**